README.md

```diff
@@ -57,12 +57,14 @@ docs/formatter.rb linguist-documentation=false
 
 Alternatively, you can use Vim and Emacs style modelines to set the language for a single file. Modelines can be placed anywhere within a file and are respected when determining how to syntax-highlight a file on GitHub.com
```

Comment:
Maybe changing `Vim and Emacs style modelines` to `Vim or Emacs style modelines` would make it more clear?